storage/src/main/java/tech/pegasys/teku/storage/store/Store.java

```diff
@@ -96,6 +94,7 @@
   final Map<Bytes32, SignedBeaconBlock> blocks;
   final CachingTaskQueue<SlotAndBlockRoot, BeaconState> checkpointStates;
   final Map<UInt64, VoteTracker> votes;
+  UInt64 highestVotedValidatorIndex;
```

Comment:
it seems wierd that this is only set in the constructor, but it can be final if it is, and wouldnt need the readlocks... maybe I'm just missing something...

Reply:
Also also set by the updater which is split into a separate class (which is also why it's package private).  It got too big to manage the updater and Store in a single class so they were split a while back.